Source/UI/NewUI/PaymentForm/Factory/Theme/BillingForm/Theme+BillingInput.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Theme+BillingInput.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 10/08/2022.
+//
+
+import UIKit
+
+public extension Theme {
+    
+    /// Theme generated Billing Input Style
+    struct ThemeBillingInput: CellTextFieldStyle {
+        public var textfield: ElementTextFieldStyle
+        public var isMandatory: Bool
+        public var backgroundColor: UIColor = .clear
+        public var title: ElementStyle?
+        public var mandatory: ElementStyle?
+        public var hint: ElementStyle?
+        public var error: ElementErrorViewStyle?
+    }
+    
+    /// Create a Billing Input from provided themed components
+    func buildBillingInput(textField: ThemeTextField,
+                           isMandatory: Bool,
+                           title: ThemeTitle,
+                           mandatory: ThemeMandatory? = nil,
+                           subtitle: ThemeSubtitle? = nil,
+                           error: ThemeError? = nil) -> ThemeBillingInput {
+        ThemeBillingInput(textfield: textField,
+                          isMandatory: isMandatory,
+                          title: title,
+                          mandatory: mandatory,
+                          hint: subtitle,
+                          error: error)
+    }
+    
+    /// Create a Billing Input from provided content
+    func buildBillingInput(text: String,
```

Comment:
@alex-ioja-yang-cko do we have any unit tests for all these build methods ?

Reply:
I haven't unit tested anything so far as we haven't tried implementing this. Once we do, our default parsing may change (ie: we shouldn't use this color there, we should use that one instead....). So as I expect changes to follow before this is out, might make sense to first implement its first use case without customisation and get Alex's feedback on the current mapping. 

Once he's happy with the mapping, can write unit tests to enforce and protect behaviour, that will be pretty straight forward (tests in format _"Given this **Theme** and this **input** is the resulting component mapped as expected"_)

Reply:
@alex-ioja-yang-cko lets add something in backlog for adding unit test to the wrapper we created otherwise we could introduce new bugs which are unexpected for us.

Reply:
[Jira task](https://checkout.atlassian.net/browse/PIMOB-1386)

[First mention of Jira task](https://github.com/checkout/frames-ios/pull/245#discussion_r944416665) 😬